### PR TITLE
[4.3] Update MediaField.php

### PR DESCRIPTION
**Pull request by brianteeman**
Grammar corrections and typos in code comment

Code Review only

### CODE REVIEW

File: libraries/src/Form/Field/MediaField.php [hunk 1]
```diff
@@ -310,8 +310,8 @@ public function getLayoutData()
             /**
              * This is the case where a folder is configured in directory attribute of the form field. The directory needs
              * to be a relative folder of the folder configured in Path to Images Folder config option of Media component.
-             * Same with a already stored local image above, we need to validate and make sure top level folder is one of the directory
-             * configured in filesystem local plugin
+             * Same with an already stored local image above, we need to validate and make sure top level folder is one of the 
+             * directories configured in tje filesystem local plugin.
```
Comment (sandewt) [inline libraries/src/Form/Field/MediaField.php]:
```suggestion
             * directories configured in the filesystem local plugin.
```

Comment (brianteeman) [reply libraries/src/Form/Field/MediaField.php]: oops

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on f88174cb6611b19101f64068fd73fafee43fc4ae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40617">issues.joomla.org/tracker/joomla-cms/40617</a>.</sub>

File: libraries/src/Form/Field/MediaField.php [hunk 2]
```diff
@@ -310,8 +310,8 @@ public function getLayoutData()
             /**
              * This is the case where a folder is configured in directory attribute of the form field. The directory needs
              * to be a relative folder of the folder configured in Path to Images Folder config option of Media component.
-             * Same with a already stored local image above, we need to validate and make sure top level folder is one of the directory
-             * configured in filesystem local plugin
+             * Same with an already stored local image above, we need to validate and make sure top level folder is one of the
```
Comment (richard67) [inline libraries/src/Form/Field/MediaField.php]:
```suggestion
             * Same with an already stored local image above, we need to validate and make sure top level folder is one of the
```
Fix PHPCS failing in Drone.

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 6d742af77f8b7a6269cd3a71df4e05102c115d66<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40617">issues.joomla.org/tracker/joomla-cms/40617</a>.</sub>

File: libraries/src/Form/Field/MediaField.php [hunk 3]
```diff
@@ -310,8 +310,8 @@ public function getLayoutData()
...
```
Comment (Quy) [inline libraries/src/Form/Field/MediaField.php]:
```suggestion
             * Same with an already stored local image above, we need to validate and make sure the top level folder is one of the
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1c31f95f4ed448b22b55a47cb13a5f98619e934a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40617">issues.joomla.org/tracker/joomla-cms/40617</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 1c31f95f4ed448b22b55a47cb13a5f98619e934a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40617">issues.joomla.org/tracker/joomla-cms/40617</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40617">issues.joomla.org/tracker/joomla-cms/40617</a>.</sub>

**Review by wojtekLs — APPROVED** [review]
LGTM :)

**Pull request merged**